circle.yml

```diff
@@ -20,3 +20,7 @@ test:
     override:
         - cd $GS_WD && go test -v $(glide novendor)
         - cd $GS_WD && go build -a -v -tags netgo -ldflags "-w -X main.gitCommit=$(git rev-parse --short HEAD)"
+        - cd $GS_WD &&
+          cp bindata.go bindata-repo.go &&
+          make update-bindata &&
+          cmp bindata.go bindata-repo.go
```

Comment:
AFAIK `cmp` fails when the files are not equal. `bindata.go` and `bindata-repo.go` should be different since `bindata.go` was regenerated by `update-bindata`. I feel like I miss something.